src/driver/src/DefaultDriver.cpp

```diff
@@ -143,10 +144,15 @@ genny::driver::DefaultDriver::OutcomeCode doRunLogic(
 
     const auto reporter = genny::metrics::Reporter{metrics};
 
-    std::ofstream metricsOutput;
-    metricsOutput.open(options.metricsOutputFileName, std::ofstream::out | std::ofstream::trunc);
-    reporter.report(metricsOutput, options.metricsFormat);
-    metricsOutput.close();
+    for (int i = 0; i < options.metricsOutputFileNames.size(); i++) {
+        const auto& fileName = options.metricsOutputFileNames[i];
+        const auto& format = options.metricsFormats[i];
```

Comment:
relying on ordering here is curious - related to above maybe type should be `std::ordered_map<string,string>` with keys being ouput type and values being file names?